netaddr.go

```diff
@@ -1163,7 +1163,12 @@ type IPPrefix struct {
 
 // IPPrefixFrom returns an IPPrefix with IP ip and port port.
```

Comment:
probably worth also mentioning here that zones are stripped.

// If ip has a zone, it is removed. See package docs.